src/mozmlops/templates/template_flow.py

```diff
@@ -63,9 +60,20 @@ def start(self):
             "WANDB_PROJECT": os.getenv("WANDB_PROJECT"),
         }
     )
+    # This PyPI decorator allows you to specify dependencies for running flows remotely.
+    # If your dependency tree is more complicated than importing a few things from pip,
+    # You can also make a custom docker image and use that with the @kubernetes decorator
```

Comment:
Note that even if you only have one dependency (e.g. `torch`), we'd still recommend building a docker image as torch itself is about 2.5gb and submitting such a big flow would fail (this is the reason why we created a docker image for the code-copilot).

Nothing to do here, but maybe we could have this specialized example w\o docker and one with docker somewhere in the repo, maybe an `examples` directory? Food for thought :)

Reply:
I like this idea a lot. I'm hopeful that we could eventually have a sort of template library :). I think that might even be a ticket!

Reply:
Indeed! Note that this flow will likely not run if it has torch as a dependency without docker

Reply:
Absolutely. A thing I think worth noting is that, I believe, outside of demos, it wouldn't be a realistic scenario for someone to be importing torch and not also have a relatively mature project and dependency situation. Most pip dependencies on my machine are in the 3 digit KB range, and a mature deep learning package like pytorch is most useful in production code on projects that are "big" enough that k8s makes more sense for environment construction than the PyPI decorator anyway 😂